go/vt/vtgate/evalengine/integration/main_test.go

```diff
@@ -0,0 +1 @@
+../../../../mysql/collations/integration/main_test.go
```

Comment:
@vmg, is this a way to import the content of `mysql/collations/integration/main_test.go` to this file?